src/Umbraco.Community.Contentment/DataEditors/DataList/DataSources/XmlDataListSource.cs

```diff
@@ -102,6 +113,12 @@ public IEnumerable<DataListItem> GetItems()
                         Value = value.Value
                     });
                 }
+                else
+                {
+                    _logger.Warn<string>("Did not recognize a name or a value in the node XML: " + node.OuterXml.Substring(0,1000));
+                    _logger.Info<string>($"Result of name XPATH ({NameXPath}): " + (name != null ? name.OuterXml : "null") );
```

Comment:
Mix of string interpolation and concatenation. I'd prefer string interpolation (if it's already being used).
Also wondering if it needs to be logged if the name/value isn't null?